packages/design-system-react/src/components/text/Text.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
```

Comment:
I think I can improve the tests here so they can be more comphrehensive I've created a subsequent ticket which will thoroghly test all prop https://github.com/MetaMask/metamask-design-system/issues/119

Reply:
Improved test PR here https://github.com/MetaMask/metamask-design-system/pull/122